helpers/common.js

```diff
@@ -0,0 +1,37 @@
+const OPERATION_TYPES = {
+  V3: {
+    REQUEST: 'request',
+    SEND: 'send',
+    REPLY: 'reply',
+    RECEIVE: 'receive',
+  },
+  V2: {
+    REQUEST: 'request',
+    SEND: 'publish',
+    REPLY: 'reply',
+    RECEIVE: 'subscribe',
```

Comment:
v2 did not support request/reply pattern, so we really need to stick to publish/subscribe always, no `request/reply`

what I mean is that in case of v2 it is either `publish` or `subscribe` and I believe `operation.reply()` is anyway always undefined in v2

not sure if you are familiar in v2

so in v2 spec, we had publish/subscribe from perspective of the user of the API. So if AsyncAPI document for service A had `publish` operation - then it meant that end user can send a message, and service A in fact subscribes to a message. Same around, if service A had `subscribe` operation, it meant that user can `subscribe` so the service A was in fact `publishing`. Yeah, confusing, thus we changed it

in v3 we say what API does, so `send` means API sends and `receive` means API receives - as simple as that.

`REQUEST: 'request'` should be `publish`
`REPLY: 'request'` should be `subscribe`

any anyway, they will never be use as in v2, reply is always undefined

Reply:
Thank you for detailed comment! I appreciate it.
I'll fix types.